commands/curation/curationaudit_test.go

```diff
@@ -631,6 +631,60 @@ func getTestCasesForDoCurationAudit() []testCase {
 				},
 			},
 		},
+		{
+			name:          "gradle tree - one blocked package",
+			tech:          techutils.Gradle,
+			pathToProject: filepath.Join("projects", "package-managers", "gradle", "curation-project"),
+			funcToGetGoals: func(t *testing.T) []string {
+				restoreWD := testUtils.ChangeWDWithCallback(t, "tests/testdata/projects/package-managers")
+				defer restoreWD()
+
+				curationCache, err := utils.GetCurationCacheFolderByTech(techutils.Gradle)
+				require.NoError(t, err)
```

Comment:
add comment why we need to change the wd? make sure locally running the test does not dirty the development environment